src/app/workflow/dag/state/dag.service.ts

```diff
@@ -29,7 +29,7 @@ import { DagStore } from './dag.store';
 
 @Injectable()
 export class DagService {
-  readonly style = [
+  readonly style: cytoscape.Stylesheet[] = [
```

Comment:
Why did this change?

Reply:
@coverbeck I was running into some run time errors with styling(don't remember the exact message), inquired Gary, that's his fix

Reply:
Didn't look too much into why, but this branch somehow caused a type conflict in the cytoscape style. This specific line didn't need to change, but might as well for easier debugging in the future.